taccsite_cms/settings.py

```diff
@@ -381,7 +381,7 @@ def gettext(s): return s
     # customize 'django.contrib.staticfiles'
     # SEE: https://stackoverflow.com/q/57921970/11817077
     # 'django.contrib.staticfiles',
-    'taccsite_cms.django.contrib.staticfiles_custom',
+    'taccsite_cms.django.contrib.staticfiles_custom.apps.TaccStaticFilesConfig',
```

Comment:
Thanks.

<details>That config is annoying to have lying around. If I didn't worry about devs loading source CSS from static, I wouldn't need it. I'll revisit</details>